draft-ietf-webtrans-http3.md

```diff
@@ -329,6 +329,20 @@ requests for new WebTransport sessions.  A GOAWAY frame does not affect data
 streams for existing WebTransport sessions; those can continue to be opened
 even after the GOAWAY frame has been sent or received.
 
+To drain a WebTransport session, either endpoint can send a
+DRAIN_WEBTRANSPORT_SESSION capsule.  An endpoint SHOULD NOT initiate new
+WebTransport streams or datagram flows after sending or receiving a
+DRAIN_WEBTRANSPORT_SESSION capsule.  Both endpoints can continue sending data on
```

Comment:
Perhaps the same note we have about max sessions that the endpoints don't agree on how many there are so please don't try to enforce the limit? Or even just reference that?

Reply:
@ekinnear : I'm not following how the stream limit is related here?

Reply:
Sorry, 
> An endpoint SHOULD NOT initiate new WebTransport streams or datagram flows after sending or receiving a DRAIN_WEBTRANSPORT_SESSION capsule.

Do both sides agree on the ordering between a new stream coming in vs. a DRAIN_WEBTRANSPORT_SESSION capsule? If not, then we may want to note that so that people plan on having them happen in any order.